db/migrate/20230719084348_create_images.rb

```diff
@@ -0,0 +1,11 @@
+class CreateImages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :images do |t|
+      t.string :pathname
+      t.integer :size
+      t.references :users, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
[OPTIONAL]
- [ ] Since you have already set the photo column with a `string` I mean URL path of the created Image, I don't think It will be important to create another table for Images as you just did.